tests/helpers/start-app.js

```diff
@@ -3,6 +3,8 @@ import Application from '../../app';
 import Router from '../../router';
 import config from '../../config/environment';
 
+Ember.ENV.RAISE_ON_DEPRECATION = true;
```

Comment:
This ensures that our dummy app (at least pages we have acceptance tests for) have no deprecations